packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -1,7 +1,127 @@
 {{page-title "Dropdown Component"}}
 
 <h2 class="dummy-h2">Dropdown Component</h2>
+<p class="dummy-paragraph">The dropdown component has several flexible features that allow developers to build the
```

Comment:
[suggestion] I think this should not be in this PR, if you look at the `melsumner/dropdown-docs` or later branches, you'll see that this will more easily fit there. But if you don't mind fixing the merge conflicts then that's fine too.

Reply:
If you mean all the showcases, I need it to develop the components.
In any case I'll have to fix the conflicts when we'll have to merge in `melsumner/dropdown-docs`.